web/src/components/Header.js

```diff
@@ -32,6 +34,23 @@ const Header = (props) => {
     </Grid>
   );
 
+  WalletService.changeAccounts(setWallet);
+
+  const connect_button =
```

Comment:
Show only when logged in?

Reply:
With the current approach you can connect the wallet at any time, but I can change it to when the user is logged in.